crates/common/types/receipt.rs

```diff
@@ -12,7 +12,7 @@ use super::TxType;
 pub type Index = u64;
 
 /// Result of a transaction
-#[derive(Clone, Debug, PartialEq, Eq)]
+#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
```

Comment:
Deserialize and Serialize have been derived here but no custom deserialization attributes have been added to the fields "cumulative_gas_used" & "succeeded", this might lead to problems down the line as ethereum expects most values to be serialized as 0x-prefixed hex strings (which is not the default behaviour of the u64 type). Is this intentional?
For more information about the expected format of serialized receipts look at struct `RpcReceipt`

Reply:
Thanks, fixed.